scenario/src/ErrorReporter.ts

```diff
@@ -1,9 +1,12 @@
+
+import { utils } from 'ethers';
 import {ComptrollerErr, TokenErr} from './ErrorReporterConstants';
 
 export interface ErrorReporter {
   getError(error: any): string | null
   getInfo(info: any): string | null
   getDetail(error: any, detail: number): string
+  getEncodedCustomError(errorName: string, args: unknown[]): string | null
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/cloudwalk/compound-protocol/pullRequest?prid=9803552)
```suggestion
  getEncodedCustomError(errorName: string, args: unknown[]): string | null;
```